docs/en/reference/installation.rst

```diff
@@ -1,5 +1,4 @@
 Installation
 ============
 
-The installation chapter has moved to `Installation and Configuration
-<reference/configuration>`_.
+The installation chapter has moved to :doc:`Installation and Configuration <reference/configuration>`_.
```

Comment:
been moved?

Reply:
Yes, it seems to happen way back then at 2.4. Must be for search engines. I wonder if we can do some kind of 301 here.